spring-cloud-gcp-vision/src/main/java/org/springframework/cloud/gcp/vision/DocumentOcrResultSet.java

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright 2017-2019 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.cloud.gcp.vision;
+
+import java.util.Iterator;
+import java.util.List;
+
+import com.google.cloud.storage.Blob;
+import com.google.cloud.storage.Storage;
+import com.google.cloud.vision.v1.TextAnnotation;
+import com.google.protobuf.InvalidProtocolBufferException;
+
+/**
+ * Represents the parsed OCR output for a document.
+ *
+ * @author Daniel Zou
+ */
+public class DocumentOcrResultSet {
+
+	private final Storage storageClient;
+
+	private final List<Blob> pageBlobs;
+
+	public DocumentOcrResultSet(List<Blob> pages, Storage storageClient) {
+		this.pageBlobs = pages;
+		this.storageClient = storageClient;
```

Comment:
Why do we have `storageClient` here? It seems it is not used anywhere.

Reply:
Nice catch. Done.